components/Tabs/index.jsx

```diff
@@ -9,11 +9,13 @@ export default class Tabs extends Component {
   static propTypes = {
     hash: PropTypes.string,
     items: PropTypes.arrayOf(PropTypes.element).isRequired,
-    pathname: PropTypes.string.isRequired
+    pathname: PropTypes.string.isRequired,
+    theme: PropTypes.oneOf(['default', 'kununuNav'])
```

Comment:
The current default theme is also used by kununu (insights) so I don't think naming the new design kununuNav will be clear.

What about block?